docs/source/_topic_map.yml

```diff
@@ -25,8 +25,10 @@ Topics:
     Topics:
       - Name: Overview
         File: index
-      - Name: Managing Minishift
-        File: managing-minishift
+      - Name: Basic commands
```

Comment:
Please use title casing: "Basic Commands".

Reply:
Ahh, of course. Will fix